### PR TITLE
Create image-sizes.md

### PR DESCRIPTION
**Pull request by brentmackinnon**
### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.

### Changes proposed in this Pull Request:

migrates [image sizing doc](https://woo.com/wp-admin/post.php?post=8734950&action=edit) from woo.com

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. this is a standalone markdown file that shouldn't affect any other code, and should only be at risk of failing linting. 
2. make sure the doc looks alright before merging.
3.

### Changelog entry

-   [ ] Automatically create a changelog entry from the details below.

<details>

#### Significance

-   [x] Patch
-   [ ] Minor
-   [ ] Major

#### Type

-   [ ] Fix - Fixes an existing bug
-   [ ] Add - Adds functionality
-   [ ] Update - Update existing functionality
-   [ ] Dev - Development related task
-   [x] Tweak - A minor adjustment to the codebase
-   [ ] Performance - Address performance issues
-   [ ] Enhancement - Improvement to existing functionality

#### Comment 

</details>

### CODE REVIEW

**Review by lsinger — CHANGES_REQUESTED** [review]
I think this PR is a good opportunity to also move over the Wiki page about thumbnail regeneration over to the docs folder. The current version of the image sizing article links to the Wiki article, and as we want to move those articles over anyway, we shouldn't introduce links to it now that we will soon have to fix.

**Comment by brentmackinnon** [comment]
great catch, leif! thank you.

**Comment by brentmackinnon** [comment]
@lsinger I think this one is ready for you to take another look.

**Comment by lsinger** [comment]
> @lsinger I think this one is ready for you to take another look.

Thanks for the heads up, @brentmackinnon. In the future, ideally (also) use the "re-request review" button to ensure the PR comes up in my [list of PRs to review](https://github.com/pulls/review-requested): 

Screenshot 2023-12-18 at 12 59 24

(No need to do that now with this PR.)

File: docs/theme-development/image-sizes.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ Store owners will still be able to control aspect ratio and cropping (see below)
...
+The customizer houses the options which control thumbnails in WooCommerce. ![Settings in the customizer](https://woocommerce.files.wordpress.com/2017/12/imagefeature.png?w=712) If the theme is declaring image sizes, the top section will be hidden and only the cropping option will be visible. Changing the cropping option, or widths, will update the preview on the right side to show how things will look. Changes will not be visible to customers until the customizer is ‘published’ and [the thumbnails have been regenerated to the new dimensions](/docs/theme-development/thumbnail-image-regeneration.md). The thumbnail cropping section in the customizer allows store owners to select one of three cropping ratio settings for images in the catalog:
```
Comment (lsinger) [inline docs/theme-development/image-sizes.md:35]: I'm not sure if an absolute link to `/docs/...md` will work once the markdown has been converted and published on the site. Pinging @rrennick for his expertise here -- is this the way to link between documents and if now, which way is?

**Comment by lsinger** [comment]
Closing and re-opening PR to see if that helps the Playwright E2E test checks get started: 

Screenshot 2023-12-18 at 13 11 12

**Pull request merged**